# WorldWideCoffe Global Expansion Analysis Report

## Sections:
1. Introduction
2. Data
3. Methodology
4. Results
5. Discussion
6. Conclusion

# 1. Introduction


WorldWideCoffee (WWC) is a fast growing startup coffeeshop company established in Eeastern Europe (Ankara, Turkey). After successful launch in the head country, it is planning to expand globally. Its main competitor globally is Starbucks. WWC is currently, considering different strategies for growth.

This Report is discussing and suggesting a growth strategy for WWC, mainly focusing on competitor analysis.



# 2. Data
Report will focus on capital cities that has at least one Starbucks shop. Data that is used in this report consists of a list of national capitals, list of starbucks shops, and information about Starbucks shops in these cities. Summary of the sour the data, and references are mentioned below:

1. List of Starbucks Shops -  https://en.wikipedia.org/wiki/Starbucks#Locations
2. List of National Capitals - https://en.wikipedia.org/wiki/List_of_national_capitals
3. FourSquare - https://foursquare.com/developers/apps


# 3. Methodology

Main target market for WWC are current unsatisfied Starbukcs customers (major competitor - global brand). That's why this report will focus on competitive analysis, and make suggestions for global expansion based on that.

The strategy company is going to use is summarized below:
1. WWC will consider the capital cities of countries throughout the world, where Starbucks has at least one branch.
2. Based on the data available online, WWC will analyze the average ratings of Starbucks shops in each city, and cluster them into 3 groups: Low, Medium, High.
3. It will choose one of the cities in the low cluster group, and expand to that country.
4. Location of the shop will be decided based on the in city ratings of Starbucks shops, ie, closer to the shops with lowest ratings.

# 4. Results

The cities and countries considered in this study (ones that include at least one Starbucks shop) are mentioned in the tabel below. Description of the data:

* AvgRatings - Average rating of all starbucks shops in the city
* Ratings - a dictionary containing, the id of a specific shop from foursquare api, and its corresponding rating


In [3]:
import pandas as pd
starbucks_shops = pd.read_csv("Ratings_final.csv", index_col=0)
starbucks_shops

Capital               Country   Latitude   Longitude  \
0             Abu Dhabi  United Arab Emirates  23.997644   53.643910   
1                 Amman                Jordan  31.951569   35.923963   
2             Amsterdam           Netherlands  52.374540    4.897976   
3      Andorra la Vella               Andorra  42.506939    1.521247   
4                Ankara                Turkey  39.921522   32.853793   
5                Athens                Greece  37.984149   23.727984   
6                  Baku            Azerbaijan  40.375443   49.832675   
7   Bandar Seri Begawan                Brunei   4.889545  114.941757   
8               Bangkok              Thailand  13.753893  100.816080   
9               Beijing                 China  39.906217  116.391276   
10               Beirut               Lebanon  33.895920   35.478430   
11             Belgrade                Serbia  44.817813   20.456897   
12               Berlin               Germany  52.517037   13.388860   
13                 Bern           Switzerland  46.948271    7.451451   
14               Bogotá              Colombia   4.598077  -74.076103   
15             Brasília                Brazil -10.333333  -53.200000   
16           Bratislava              Slovakia  48.135908   17.159744   
17             Brussels               Belgium  50.846557    4.351697   
18            Bucharest               Romania  44.436141   26.102720   
19             Budapest               Hungary  47.498382   19.040471   
20         Buenos Aires             Argentina -34.607562  -58.437076   
21                Cairo                 Egypt  30.048819   31.243666   
22             Canberra             Australia -35.297591  149.101268   
23           Copenhagen               Denmark  55.686724   12.570072   
24                 Doha                 Qatar  25.285633   51.526416   
25               Dublin               Ireland  53.349764   -6.260273   
26       Guatemala City             Guatemala  14.622233  -90.518519   
27                Hanoi               Vietnam  21.029450  105.854444   
28             Helsinki               Finland  60.171320   24.941457   
29              Jakarta             Indonesia  -6.175394  106.827183   
..                  ...                   ...        ...         ...   
47              Nicosia                Cyprus  35.180282   33.373696   
48            Nursultan            Kazakhstan  43.086368   77.120336   
49           Oranjestad                 Aruba  12.526874  -70.035685   
50                 Oslo                Norway  59.913330   10.738970   
51               Ottawa                Canada  45.260398  -75.808238   
52          Panama City                Panama   8.971449  -79.534180   
53                Paris                France  48.856610    2.351499   
54           Phnom Penh              Cambodia  11.568271  104.922443   
55        Port of Spain   Trinidad and Tobago  10.657268  -61.518017   
56               Prague        Czech Republic  50.087465   14.421254   
57            Cape Town          South Africa -33.928992   18.417396   
58                Rabat               Morocco  34.022405   -6.834543   
59               Riyadh          Saudi Arabia  24.631969   46.715065   
60                 Rome                 Italy  41.894802   12.485338   
61             San José            Costa Rica   9.932543  -84.079578   
62             San Juan           Puerto Rico  18.465299  -66.116666   
63         San Salvador           El Salvador  13.698994  -89.191425   
64             Santiago                 Chile -33.437797  -70.650445   
65                Seoul           South Korea  37.566679  126.978291   
66            Singapore             Singapore   1.340853  103.878447   
67                Sofia              Bulgaria  42.697863   23.322179   
68            Stockholm                Sweden  59.325117   18.071093   
69               La Paz               Bolivia -16.495637  -68.133635   
70               Taipei                Taiwan  25.037520

### K-means

After preparing the table above, K-means algorithm was used to divide cities into 3 categories below:

1. Cities with **High Ratings**
2. Cities with **Medium Ratings**
3. Cities with **Low Ratings**


#### Cities with High Rating Starbucks stores:

In [6]:
pd.read_csv('high_rated_cities.csv', index_col=0)

Cluster Labels         Capital               Country   Latitude  \
0                1       Abu Dhabi  United Arab Emirates  23.997644   
1                1           Amman                Jordan  31.951569   
8                1         Bangkok              Thailand  13.753893   
16               1       Bucharest               Romania  44.436141   
19               1           Cairo                 Egypt  30.048819   
21               1            Doha                 Qatar  25.285633   
23               1  Guatemala City             Guatemala  14.622233   
24               1           Hanoi               Vietnam  21.029450   
28               1     Kuwait City                Kuwait  29.379709   
30               1          Lisbon              Portugal  38.707751   
34               1          Manama               Bahrain  26.223504   
39               1          Moscow                Russia  55.750446   
43               1         Nicosia                Cyprus  35.180282   
48               1     Panama City                Panama   8.971449   
53               1          Riyadh          Saudi Arabia  24.631969   
54               1        San José            Costa Rica   9.932543   
60               1           Sofia              Bulgaria  42.697863   
68               1      Willemstad               Curaçao  12.109124   

     Longitude  AvgRatings                                            Ratings  
0    53.643910         8.4                  {'53089160498e151fa7f9a9f5': 8.4}  
1    35.923963         9.1  {'4bb624ee46d4a593f8c0c5c0': 9.1, '59fcc6bb286...  
8   100.816080         8.3  {'56c6a4c1498e6dd6d1bddb91': 8.3, '5a3c5fe36a5...  
16   26.102720         8.4  {'5491a88f498e12011af1a3a7': 8.4, '54eed7ad498...  
19   31.243666         9.0  {'4bdf04120ee3a59313ac32b0': 9.0, '5ac227211f8...  
21   51.526416         8.3  {'4ce11c2c94c3b60c68bd74ea': 8.3, '4b9d2789f96...  
23  -90.518519         9.0  {'5628e1b1498e2c6aac7934b1': 9.0, '55a808e8498...  
24  105.854444         8.7  {'590d4a5e1108ba582673bec7': 8.7, '53d9d5e3498...  
28   47.973563         9.3  {'5644fad1498e387283e35085': 9.3, '58a025a003e...  
30   -9.136592         8.3  {'4c10f43281e976b08deb0feb': 8.3, '4b3fc213f96...  
34   50.582244         9.2  {'5c29d763f870fd002cce4724': 9.2, '4bdc05e263c...  
39   37.617494         8.6  {'4f3b52bfe4b0ce0258fcafd4': 8.6, '4ba755a8f96...  
43   33.373696         8.3  {'4d1f501486e15481311fb13a': 8.3, '4bb2196ff96...  
48  -79.534180         8.6  {'5655d0a138fa700382e6516c': 8.6, '57e542b9498...  
53   46.715065         9.3  {'566db885498e8b884eecff76': 9.3, '5a378e55d76...  
54  -84.079578         8.7  {'561da3f7498ee3b75b76f2cb': 8.7, '56059049498...  
60   23.322179         8.4  {'4ba0dbd1f964a520b58237e3': 8.4, '4b6d8a4cf96...  
68  -68.931655         8.4  {'4e480b8381305ff36220a929': 8.4, '518d4c49498...

#### Cities with Medium Rating Starbucks stores:

In [7]:
pd.read_csv('medium_rated_cities.csv', index_col=0)

Cluster Labels              Capital              Country   Latitude  \
2                2            Amsterdam          Netherlands  52.374540   
3                2     Andorra la Vella              Andorra  42.506939   
4                2               Ankara               Turkey  39.921522   
5                2               Athens               Greece  37.984149   
6                2                 Baku           Azerbaijan  40.375443   
7                2  Bandar Seri Begawan               Brunei   4.889545   
9                2              Beijing                China  39.906217   
10               2               Beirut              Lebanon  33.895920   
14               2           Bratislava             Slovakia  48.135908   
15               2             Brussels              Belgium  50.846557   
17               2             Budapest              Hungary  47.498382   
18               2         Buenos Aires            Argentina -34.607562   
20               2           Copenhagen              Denmark  55.686724   
25               2             Helsinki              Finland  60.171320   
29               2                 Lima                 Peru -12.062107   
33               2               Madrid                Spain  40.416705   
36               2          Mexico City               Mexico  19.432601   
37               2               Monaco               Monaco  43.731142   
38               2           Montevideo              Uruguay -34.905904   
40               2               Muscat                 Oman  23.512550   
42               2            New Delhi                India  28.614179   
45               2           Oranjestad                Aruba  12.526874   
50               2        Port of Spain  Trinidad and Tobago  10.657268   
55               2             San Juan          Puerto Rico  18.465299   
56               2         San Salvador          El Salvador  13.698994   
57               2             Santiago                Chile -33.437797   
62               2               Taipei               Taiwan  25.037520   
64               2               Vienna              Austria  48.208354   
65               2               Warsaw               Poland  52.231924   

     Longitude  AvgRatings                                            Ratings  
2     4.897976         8.0  {'529634bc11d2ab5263ab848c': 8.0, '50056329e4b...  
3     1.521247         7.9  {'5745a9cc498e4f0f9a79db85': 7.9, '595a40052bf...  
4    32.853793         8.1  {'4f25511ce4b04f6e6a110d1b': 8.1, '51386bcbe4b...  
5    23.727984         7.9  {'4b55dc00f964a52046f327e3': 7.9, '4b9f7ab4f96...  
6    49.832675         8.0  {'5698c3bc498eac8f3804d4df': 8.0, '576d0a8c498...  
7   114.941757         7.6  {'52fc9a45498e79749d33f0d0': 7.6, '4c4d13aff5a...  
9   116.391276         7.7  {'4be7eebbee96c9289b6efdbf': 7.7, '5b373a893d4...  
10   35.478430         7.8  {'4bb72c4c2f70c9b6c8578630': 7.8, '4c8d1177c37...  
14   17.159744         7.5  {'5799a6f3498e27299438fd56': 7.5, '5745d8fa498...  
15    4.351697         8.0  {'4e00a764b3ad0e090a6ff1ac': 8.0, '537e0ffb498...  
17   19.040471         7.7  {'4f89d5f24fc69a9a7d07ed32': 7.7, '578dc1f8cd1...  
18  -58.437076         7.7  {'4b8d092bf964a52036e532e3': 7.7, '4d11ed9a7a8...  
20   12.570072         7.6  {'55ac9af9498e117d7f2a1f0a': 7.6, '53ad114b498...  
25   24.941457         7.5  {'5238643c11d2c1029cc5e106': 7.5, '5631da7f498...  
29  -77.036526         8.2  {'52462e2911d2d573d4b3278e': 8.2, '51223a58e4b...  
33   -3.703582         7.6  {'4b9b5750f964a520790136e3': 7.6, '4b66c179f96...  
36  -99.133342         7.6  {'4bb2455ba32876b0a95f00fe': 7.6, '514e5790e4b...  
37    7.419758         7.9  {'5231f448498e480466833be9': 7.9, '57a9e413498...  
38  -56.191357         7.4  {'5ab2b3dba92d9801c7544674': 7.4, '5bb14088446...  
40   58.551402         8.1  {'5b8b7203419a9e002cfefc1e': 8.1, '4c3596dca0c...  
42   77.202266         8.2  {'5113713fe4b09f5a3d6a7590': 8.2, '5328522849

#### Cities with Low Rating Starbucks stores:

In [10]:
pd.read_csv('low_rated_cities.csv', index_col=0)

Cluster Labels       Capital         Country   Latitude   Longitude  \
11               0        Berlin         Germany  52.517036   13.388860   
12               0          Bern     Switzerland  46.948271    7.451451   
13               0        Bogotá        Colombia   4.598077  -74.076103   
22               0        Dublin         Ireland  53.349764   -6.260273   
26               0       Jakarta       Indonesia  -6.175394  106.827183   
27               0  Kuala Lumpur        Malaysia   3.151664  101.694303   
31               0        London  United Kingdom  51.507322   -0.127647   
32               0    Luxembourg      Luxembourg  49.815868    6.129675   
35               0        Manila     Philippines  14.590622  120.979970   
41               0        Nassau         Bahamas  25.078346  -77.338333   
44               0     Nursultan      Kazakhstan  43.086368   77.120336   
46               0          Oslo          Norway  59.913330   10.738970   
47               0        Ottawa          Canada  45.260398  -75.808238   
49               0         Paris          France  48.856610    2.351499   
51               0        Prague  Czech Republic  50.087465   14.421254   
52               0         Rabat         Morocco  34.022405   -6.834543   
58               0         Seoul     South Korea  37.566679  126.978291   
59               0     Singapore       Singapore   1.340853  103.878447   
61               0     Stockholm          Sweden  59.325117   18.071093   
63               0         Tokyo           Japan  35.682839  139.759455   
66               0    Washington   United States  38.895009  -77.036563   
67               0    Wellington     New Zealand -41.288747  174.777209   

    AvgRatings                                            Ratings  
11        6.90  {'4adf77c3f964a520df7a21e3': 6.9, '4adf768af96...  
12        7.20  {'4b7687ddf964a520b4502ee3': 7.2, '4b61bad2f96...  
13        6.70  {'58d0742a951e7d532bac7fb2': 6.7, '54f83f18498...  
22        7.30  {'5290d704498e2d5c6000a181': 7.3, '4af1ea70f96...  
26        7.30  {'51047707e4b0059ce06c2cf0': 7.3, '5783a0ff498...  
27        6.80  {'5902d2f4646e387f2b2d705c': 6.8, '538daac6498...  
31        6.80  {'502904745dd7750e9d63bc17': 6.8, '4b77eb6ff96...  
32        7.20  {'567eae1e498eab68b90d203b': 7.2, '568f8115498...  
35        7.10  {'4cd7475ab6962c0fc4b72f96': 7.1, '4bda44e6a8d...  
41        6.50  {'4bcf7dd041b9ef3b078cf8e5': 6.5, '4ccafe8eba0...  
44        6.90  {'5782361c498e2299113fdb5a': 6.9, '578b98d8498...  
46        6.40  {'519e5ee1498e59575ef2ae8c': 6.4, '5279f00d11d...  
47        6.50  {'4b2b9f33f964a5205eb824e3': 6.5, '4b15a589f96...  
49        7.00  {'4d9da49ec97a236afb68c199': 7.0, '4b73c62cf96...  
51        6.24  {'4b46db8bf964a520f12826e3': 7.8, '5c1f5190dbd...  
52        7.20  {'59da90824420d82388d1b745': 7.2, '59e4731a624...  
58        6.20  {'56f573d5498e64a809fc4527': 6.2, '4b5bb69af96...  
59        6.40  {'4cea08b7e888f04d1f404b6b': 6.4, '4b062a5cf96...  
61        7.00  {'54bcfcc3498ea0b8417f2eb2': 7.0, '54648a88498...  
63        6.70  {'4b85ba2bf964a520666f31e3': 6.7, '597185d6080...  
66        6.90  {'4a6718ecf964a520e7c81fe3': 6.9, '4a54d2f3f96...  
67        6.60  {'4b058778f964a5204f9422e3': 6.6, '4b233adbf96...

# 5. Discussion

Some observations from the results:

* Cities with average rating of 7.3 are in the lower rated cluster.
* Starbucks shops with lower ratings are mainly located in developed countries/cities. 

From the table above Seoul (South Korea) and Prague (Czech Republic) looks like a very good candidate for expansion. They have the lowest rated starbucks stores, and offer a good customer base both in terms of local population, and tourists visiting the city.


# 6. Conclusion

In conclusion, WWC should consider Seoul, and Prague for potential expansion.

Furthermore, in each city they can analyze the starbucks shops, and choose a location closer to a Starbucks shop that has the lowest rating.

The dictionary in the **'Ratings'** Column of the above table provides the required info for that analysis.